MainWindow.xaml.cs

```diff
@@ -219,5 +220,47 @@ private void UpdateEditWindows()
                 _edit_metadata_window.Content = user_control;
             }
         }
+
+        #region Test
+        public void TestInputPack(Data.DynamicInputPack inputPack)
```

Comment:
Unless you plan on adding other checks, this function could be more explicit.  Maybe call it `CheckTexturePathes`